code/modules/media/jukebox.dm

```diff
@@ -405,20 +427,26 @@ var/global/loopModeNames=list(
 		attack_hand(user)
 
 /obj/machinery/media/jukebox/emag(mob/user)
-	current_song = 0
 	if(!emagged)
-		playlist_id = "emagged"
-		last_reload=world.time
-		playlist=null
-		loop_mode = JUKEMODE_SHUFFLE
-		emagged = 1
-		playing = 1
 		user.visible_message("<span class='warning'>[user.name] slides something into the [src.name]'s card-reader.</span>","<span class='warning'>You short out the [src.name].</span>")
-		update_icon()
-		update_music()
-		return 1
+		wires.CutWireIndex(JUKE_CONFIG)
 	return
 
+/obj/machinery/media/jukebox/proc/short()
```

Comment:
An informal strawpoll in the thread showed that twice as many people thought it should be possible to hack a jukebox to get the syndicate playlist than those who thought it should stay as it is.

However, of those people who wanted it to be hackable, the overwhelming majority wanted it to be a logic puzzle so that it would require some work. Therefore, I will set to work making it a more difficult hack.